conjur/interface/credentials_store_interface.py

```diff
@@ -6,43 +6,52 @@
 This class describes a shared interface for accessing user credentials
 """
 
+# 3rd parties
+import abc
+
 # Internals
 from conjur.data_object import CredentialsData
 
-# TODO use ABC formal interface to make sure inherit classes implement interface
+
 # pylint: disable=unnecessary-pass
```

Comment:
you can remove this pylint because we removed `pass`

Reply:
done